src/users/store/reducers/users.reducer.ts

```diff
@@ -29,7 +29,41 @@ export function reducer(
     }
 
     case fromUsers.LOAD_USERS_SUCCESS: {
-      const userList = action.payload.users;
+      const payload = action.payload.users;
+
+      const userListPayload = payload.map((item) =>
+        Object.assign({}, item, { selected: false })
```

Comment:
this is old way to do stuff 
you can u spreader operator 
{...item, selected: false}